content/en/boilerplates/cve-2020-007-configmap.md

```diff
@@ -0,0 +1,37 @@
+---
+---
+1. Create a config map by downloading [custom-bootstrap-runtime.yaml](/news/security/istio-security-2020-007/custom-bootstrap-runtime.yaml). Update `global_downstream_max_connections` in the config map according to the number of concurrent connections needed by individual gateway instances in your deployment. Once the limit is reached, Envoy will start rejecting tcp connections.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f custom-bootstrap-runtime.yaml
+    {{< /text >}}
+
+1. Patch the ingress gateway deployment to use the above configuration. Download [gateway-patch.yaml](/news/security/istio-security-2020-007/gateway-patch.yaml) and apply it using the following command.
+
+    {{< text bash >}}
+    $ kubectl --namespace istio-system patch deployment istio-ingressgateway --patch "$(cat gateway-patch.yaml)"
+    {{< /text >}}
+
+1. Confirm that the new limits are in place.
+
+    {{< text bash >}}
+    $ ISTIO_INGRESS_PODNAME=$(kubectl get pods -l app=istio-ingressgateway -n istio-system  -o jsonpath="{.items[0].metadata.name}")
+    $ kubectl --namespace istio-system exec -i -t  "${ISTIO_INGRESS_PODNAME}" -c istio-proxy -- curl -sS http://localhost:15000/runtime
+
+    {
+    "entries": {
+    "overload.global_downstream_max_connections": {
+    "layer_values": [
+    "",
+    "250000",
+    ""
+    ],
+    "final_value": "250000"
+    }
+    },
+    "layers": [
+    "static_layer_0",
+    "admin"
+    ]
+    }
+    {{< /text >}}
```

Comment:
There was indenting in the old version that is lost in this version.  Otherwise, it's all LGTM!